libs/domains/services/feature/src/lib/pods-metrics/pods-metrics.tsx

```diff
@@ -0,0 +1,64 @@
+import { LoaderSpinner, TablePrimitives } from '@qovery/shared/ui'
+import { useMetrics } from '../hooks/use-metrics/use-metrics'
+
+const { Table } = TablePrimitives
+
+export interface PodsMetricsProps {
+  environmentId: string
+  serviceId: string
+}
+
+export function PodsMetrics({ environmentId, serviceId }: PodsMetricsProps) {
+  const { data = [], isLoading } = useMetrics({ environmentId, serviceId })
+
+  if (isLoading) {
+    return (
+      <div className="w-full flex items-center justify-center">
+        <LoaderSpinner className="w-6" />
+      </div>
```

Comment:
Instead of use this one, I think we can use `<Skeleton />` for each cell, should be more elegant, wdyt ?

Reply:
https://github.com/Qovery/console/pull/796/commits/3c8bfd06afee79fd63c60123fb498b75f6bba737